geopandas/base.py

```diff
@@ -135,6 +135,30 @@ def area(self):
         ``GeoSeries``."""
         return _delegate_property("area", self)
 
+    @property
+    def crs(self):
+        """Returns a ``pyproj.CRS``"""
+        return self._crs
+
+    @crs.setter
+    def crs(self, value):
+        """Sets the value of the crs.
+
+        The value can be anything accepted
+        by ``pyproj.CRS.from_user_input``.
+            - PROJ string
```

Comment:
Can you dedent this? (rst does not need indentation for lists, only a blank line)

Reply:
Sure